jobs/nginx/11ty-website.hcl

```diff
@@ -0,0 +1,46 @@
+job "11ty-website" {
+  datacenters = ["aperture"]
+  type = "service"
+
+  group "nginx-11ty-website" {
+    count = 1
+
+    network {
+      port "http" {
+        to = 80
+      }
+    }
+
+    service {
+      port = "http"
+
+      check {
+        type = "http"
+        path = "/"
+        interval = "10s"
+        timeout = "2s"
+      }
+
+      tags = [
+        "traefik.enable=true",
+        "traefik.http.routers.nginx-11ty-website.rule=Host(`canary.redbrick.dcu.ie`)",
```

Comment:
For now, this is okay. When the site is released though, this tag will be moved into a `canary_tags` block.